src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
```suggestion
    <ul class="stars stars--0">
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
      <li class="stars__star"></li>
    </ul>
```
It would be better 😉